Dockerfile

```diff
@@ -120,11 +120,18 @@ RUN cd /tmp && \
     tar xzf /builds/worker/Downloads/android-sdk_r24.3.4-linux.tgz --directory=/builds/worker || true && \
     unzip -qq -n /builds/worker/Downloads/sdk-tools-linux-4333796.zip -d /builds/worker/android-sdk-linux/ || true && \
     /builds/worker/android-sdk-linux/tools/bin/sdkmanager platform-tools "build-tools;28.0.3" && \
+    # upgrade the builtin setuptools
     pip install setuptools -U && \
-    pip install mozdevice==3.0.5 && \
+    pip3 install setuptools -U && \
+    # pips used by scripts in this docker image
```

Comment:
you need a trailing escape after the comment in order to keep the pips all in the overall command line.

Reply:
The Dockerfile builds locally.

https://docs.docker.com/engine/reference/builder/#format indicates that "Line continuation characters are not supported in comments."

Reply:
The comments seem to be dropped. Command run when building:

Step 23/25 : RUN cd /tmp &&     chmod +x /usr/local/bin/generic-worker &&     chmod +x /usr/local/bin/livelog &&     chmod +x /usr/local/bin/tooltool.py &&     chmod +x /usr/local/bin/entrypoint.* &&     chmod +x /builds/taskcluster/script.py &&     mkdir /root/.android &&     touch /root/.android/repositories.cfg &&     tar xzf /builds/worker/Downloads/node-v8.11.3-linux-x64.tar.gz -C /usr/local --strip-components 1 &&     node -v &&     npm -v &&     tar xzf /builds/worker/Downloads/android-sdk_r24.3.4-linux.tgz --directory=/builds/worker || true &&     unzip -qq -n /builds/worker/Downloads/sdk-tools-linux-4333796.zip -d /builds/worker/android-sdk-linux/ || true &&     /builds/worker/android-sdk-linux/tools/bin/sdkmanager platform-tools "build-tools;28.0.3" &&     pip install setuptools -U &&     pip3 install setuptools -U &&     pip install google-cloud-logging &&     pip3 install google-cloud-logging &&     pip install mozdevice==3.0.5 &&     pip3 install mozdevice==3.0.5 &&     pip install zstandard==0.11.1 &&     pip3 install zstandard==0.11.1 &&     rm -rf /tmp/* &&     rm -rf /var/lib/apt/lists/* &&     rm -rf /builds/worker/Downloads/* &&     chown -R root:worker /builds &&     chmod 775 /builds